RoadMap_Movie/RoadMap_Movie/MainModule/View/MainScreenViewController.swift

```diff
@@ -208,9 +179,18 @@ extension MainScreenViewController: UITableViewDataSource, UITableViewDelegate {
     }
 
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        let movieId = movies?.movies[indexPath.row].id
-        let detailsViewController = MovieDetailViewController()
-        detailsViewController.movieId = movieId
-        navigationController?.pushViewController(detailsViewController, animated: true)
+        presenter?.tapOnTheMovie(id: movies[indexPath.row].id)
+    }
+}
+
+// MARK: - MainScreenProtocol
+
+extension MainScreenViewController: MainScreenProtocol {
+    func success() {
+        movies = presenter?.movies ?? []
+    }
+
+    func failure(error: Error) {
+        print(error.localizedDescription)
```

Comment:
```suggestion
        print(error.localizedDescription)
```
ай я яй, давай Алерт выводить